README.md

```diff
@@ -265,6 +270,16 @@ Click to expand any of the code examples below.
 All of these can be run in a Colab notebook:
 [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/langroid/langroid/blob/main/examples/langroid_quick_examples.ipynb)
 
+We also provide a containerized version of `Langroid` repository via this [Docker Image](https://hub.docker.com/r/mohannadcse/langroid). All what you need is just setting up environment variables under `.env`. Please follow these steps to setup the container. 
+```bash
+wget https://github.com/langroid/langroid/blob/main/.env-template .env
+# add API keys as explained in https://github.com/langroid/langroid#set-up-environment-variables-api-keys-etc
+nano .env
+docker run -it  -v ./.env:/langroid/.env mohannadcse/langroid /bin/bash
```

Comment:
I ran this from inside my `langroid-examples` root folder, which already has a `.env ` file, and it should have mounted it to the image, but I get an err saying open ai key not set.

Reply:
I'll push the fix shortly, there is a problem with the path inside the container. 

<img width="1143" alt="image" src="https://github.com/langroid/langroid/assets/15859139/b545496f-bf28-4100-bcad-9ae4fbcb3f1d">